frame/contracts/src/tests.rs

```diff
@@ -4771,8 +4797,8 @@ fn deposit_limit_honors_liquidity_restrictions() {
 		assert_eq!(get_contract(&addr).total_deposit(), min_balance);
 		assert_eq!(<Test as Config>::Currency::total_balance(&addr), min_balance);
 
-		// check that the lock ins honored
-		Balances::set_lock([0; 8], &BOB, 1_000, WithdrawReasons::TRANSFER);
```

Comment:
`set_lock` does not exist anymore.
Thought the closes method would be `set_freeze`, this one only prevents actions that reduce the balance of the account.
Because now the deposit storage is held by the `hold` method, that does not reduce the balance of the account, we can't test this with `set_freeze`. Essentially, frozen balances can be held.
But we can't hold already held funds, and that's why I have chosen `hold` as the closet migration from `set_lock` in this test.